### PR TITLE
add ie8 support

### PR DESCRIPTION
**Pull request by webpatch**
**Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?**
- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the following... 

* Impact:
* Migration path for existing applications: 
* Github Issue(s) this is regarding:

**Other information**:

### CODE REVIEW

File: buildin/module.js [hunk 1]
```diff
@@ -4,16 +4,21 @@ module.exports = function(module) {
...
+        get: function() { return module.l; }
+      });
+      Object.defineProperty(module, "id", {
+        enumerable: true,
+        configurable: false,
+        get: function() { return module.i; }
+      });
+		}catch (e){
+      module["loaded"] = function() { return module.l; };
+      module["id"] = function() { return module.i; };
```
Comment (Kovensky) [inline buildin/module.js]: This changes the type of `module["loaded"]` and `module["id"]` from `typeof module.l` / `typeof module.i` to `function`.

**Comment by Kovensky** [comment]
I am surprised this passes tests; `module.js` has mixed tabs/spaces.

**Comment by sokra** [comment]
> I am surprised this passes the linter test; module.js has mixed tabs/spaces.

Yeah, maybe this folder is not linted...

**Comment by sokra** [comment]
After some discussion we decided for the following statement regarding IE8:

We don't support IE8 in webpack 2. IE8 is not supported by Microsoft and doesn't get security fixes. We think it's no responsible to allow web developers to build webapps which support IE8. They must break!

**Pull request closed without merging**